src/main/resources/spring/camel-context.xml

```diff
@@ -51,6 +65,53 @@
             </get>
         </rest>
 
+        <rest path="/administration" produces="application/octet-stream">
+            <get uri="/report/csv">
+                <route id="administration-report-csv">
+                    <setHeader headerName="CSVCorrelationID">
```

Comment:
Please create static field in `MainRouteBuilder` (there are already some examples) in order to avoid to repeat the value `CSVCorrelationID` multiple times